src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -276,7 +266,7 @@ public void initSendable(SendableBuilder builder) {
           return backMotorEncoder.getVelocity();
         },
         null);
-    builder.addBooleanProperty("beamBreakSensorOne", () -> beamBreakSensorOne.isPressed(), null);
+    builder.addBooleanProperty("beamBreakSensor", () -> !frontConveyorBeamBreakSensor.get(), null);
```

Comment:
Change name to differentiate from intake. Also add back

Reply:
Done. I removed the !s because IMO something named "sensor" should be returning the actual sensor output; I can absolutely see us spending time wondering why it's backwards and not realizing we flipped the shuffleboard output.

Reply:
Maybe rename to "front conveyor sees object" and keep the !

Reply:
Cool, that's probably a bit cleaner. I'll do the same for the other two.

Reply:
Done